### PR TITLE
millw 1.0 (new formula)

### PR DESCRIPTION
**Pull request by h2000**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by h2000** [comment]
> * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)

This formula installs an improved wrapper for the mill projects, written by a mill maintainer (lefou) and https://github.com/lihaoyi/mill is very notable.

**Comment by fxcoudert** [comment]
`mill` is very notable, as you said, but if this new formula is incompatible with it, that's not really an argument.

File: Formula/millw.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Millw < Formula
+  desc "Mill wrapper script, automatically fetch and execute mill"
+  homepage "https://github.com/lefou/millw"
+  url "https://raw.githubusercontent.com/lefou/millw/master/millw"
```
Comment (fxcoudert) [inline Formula/millw.rb]: URL should be for a released, stable version

File: Formula/millw.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  version "1.0"
+  sha256 "00528ff087739f2fc5e8b43d2f376ebb8eb30bd745bcdd69283a108968a42580"
+
+  conflicts_with "mill", :because => "millw is an improved version of the mill starter script"
+
+  resource "bashcompl" do
```
Comment (fxcoudert) [inline Formula/millw.rb]: No to using this as a separate resource. It should be packaged with the script.

**Comment by fxcoudert** [comment]
Actually, I just read the beginning of the script:

> This is a wrapper script, that automatically download mill from GitHub release pages

This makes it not suitable for Homebrew core. You can maintain it in a personal or organisation tap: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Pull request closed without merging**